awscli/examples/cloudwatch/untag-resource.rst

```diff
@@ -0,0 +1,9 @@
+**To remove one or more tags from the specified resource**
+
+The following ``untag-resource`` example removes 2 tags from the cloudwatch alarm named ``demo`` in the specified account. If the command succeeds, no output is returned. ::
```

Comment:
Same here. The line for "This command produced no output." should be moved to after the command example where the output section is.
https://aws.github.io/aws-cli/docs_styleguide.html#command-produces-no-output